crates/openapi/src/openapi.rs

```diff
@@ -261,6 +262,7 @@ Never share your secret api keys. Keep them guarded and secure.
         api_models::admin::BusinessPaymentLinkConfig,
         api_models::admin::PaymentLinkConfigRequest,
         api_models::admin::PaymentLinkConfig,
+        api_models::admin::TtlForExtendedCardInfo,
```

Comment:
This is a newtype wrapper right? We probably don't want to include this in the API spec. better to put `#[schema(value_type = u16)]` over the field in the API request struct instead.